utils/generate_types/overrides-test.d.ts

```diff
@@ -65,11 +65,14 @@ export interface FullConfig<TestArgs = {}, WorkerArgs = {}> {
 
 export type TestStatus = 'passed' | 'failed' | 'timedOut' | 'skipped' | 'interrupted';
 
-export type TestDetailsAnnotation = {
+export type TestAnnotation = {
   type: string;
   description?: string;
+  location?: Location;
 };
 
+export type TestDetailsAnnotation = Omit<TestAnnotation, 'location'>;
```

Comment:
Funnily, I'd instead do `TestAnnotation = TestDetailsAnnotation & { location?: Location }`. Should be equivalent though.